config/secrets.yml

```diff
@@ -80,6 +82,7 @@ staging:
 
 production:
   <<: *default
+  store_assets_in_file_system: false
```

Comment:
Here the value is `false` which is right. But it does not make it explicit that the alternative is storing in s3. There are alternatives to s3 too. 

I think a better key would be `store_uploaded_assets_in` with correct values being either `file_system` or `s3`.